js/StandardLibraryIntellisense.ts

```diff
@@ -222,3 +237,20 @@ function CacheOp(arguments: IArguments, cacheMiss: () => oc.TopoDS_Shape): oc.To
  * [Source](https://github.com/zalo/CascadeStudio/blob/master/js/CADWorker/CascadeStudioStandardLibrary.js)
  * @example```let box = CacheOp(arguments, () => { let box = Box(x,y,z); sceneShapes = Remove(sceneShapes, box); return box; });``` */
 function Remove(array: any[], toRemove: any): any[];
+
+const Helpers: Helpers2
+interface Helpers2 {
+  getTangentialPointOfTwoCircles: (arg: {
```

Comment:
Not 100% about the api here on this one.
So the option I've gone is that the `from` and `to` circles are needed as well as clockwise for both of them because for two circles that aren't intersection at all, there are for common tangents
![image](https://user-images.githubusercontent.com/29681384/104170671-a27d3500-5455-11eb-9e73-3a9b431ac396.png)
So setting clockwise for both of them is as if you were traveling around the first circle clockwise before following the tangent that leads you to the second circle that allows you to keep traveling around it clockwise two. Setting one to clockwise and one to counterCW is for one of the transverse tangents. Do you think there's a better way of describing intention here?